openshift-kube-apiserver/configdefault/admission_config.go

```diff
@@ -7,21 +7,13 @@ import (
 	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/kubernetes/cmd/kube-apiserver/app/options"
-	"k8s.io/kubernetes/openshift-kube-apiserver/admission/customresourcevalidation/customresourcevalidationregistration"
```

Comment:
can this commit squashed into the one which introduces `openshift-kube-apiserver/`?

Reply:
Done. Squashed with `UPSTREAM: <carry>: openshift-kube-apiserver: add kube-apiserver patches`.